src/content/docs/browser/new-relic-browser/configuration/filter-ajax-request-events.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: 'Filter AjaxRequest events'
+metaDescription: "For New Relic browser monitoring: use the app settings page to filter which network calls are recorded as AjaxRequest events."
+---
+In agent version v1211 or higher, all network requests made by a page are recorded as AjaxRequest events. Included with this change is a new deny list configuration option available in the application settings page to filter which requests record events.
```

Comment:
A few recommendations:

- Use either "version 1211 or higher" or "v1211 or higher" No need for "version v1211"
- Add a link to https://docs.newrelic.com/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes/ when you finalize the version
- Enclose AjaxRequest in backticks so it renders as code here. But not in title or metadescription because code looks odd there. :-)
One other thing to consider: "Included with this change is a new deny list" is great info for a release note, because there it's truly new! But for people reading your page going forward, calling this new could be confusing, esp 2 years later. :-) May want to consider wording second sentence something like, "You can use the deny list configuration options in the **Application settings** page to filter which requests record events."